### ISSUE TITLE
Add Anaconda support

### ISSUE THREAD

**Issue by rhyolight** [numenta/nupic#2522]
This means: 

- [ ] Create a conda recipe http://conda.pydata.org/docs/building/recipe.html
- [ ] Manually deploy conda package(s) for future CI builds
- [ ] Create Travis-CI build option for Anaconda for OS X
- [ ] Create Travis-CI build option for Anaconda for Linux
- [ ] Update builds to deploy on release to anaconda.com

**Comment by jcasner** [comment]
I would argue that we don't need to be able to run `conda install nupic` to be able to claim support for anaconda.  Those are nice to haves, not critical features.

**Comment by rhyolight** [comment]
Fair enough. Instead of this:

- [ ] Create a conda recipe http://conda.pydata.org/docs/building/recipe.html
- [ ] Manually deploy conda package(s) for future CI builds
- [ ] Update builds to deploy on release to anaconda.com

I've updated to this:

- [ ] Update wiki with instructions for Anaconda setup

**Comment by rcrowder** [comment]
Bear in mind that packages such as Anaconda (http://docs.continuum.io/anaconda/pkg-docs) currently bundle up-to Python 3.4. We require 3.5 for the NuPIC C++ extensions. Fingers crossed  Continuum add 3.5 support in the next few months.
Once we have 3.5 support, we could try 3.4. It would require all of NuPIC core and external libraries to be rebuilt specially (Z, Apr, etc.). Something I'm doing for 3.5 now, so carries over.

**Comment by rhyolight** [comment]
> PS: Read other issues, and sounds like this is related to CentOS only?

No, this is an issue across operating systems. I've received bug reports about Anaconda intermittently for the past two years. Perhaps all we need to do is prove it works in CI for OS X and Linux for now.
